master/docs/examples/git_gerrit.cfg

```diff
@@ -184,7 +184,15 @@ def gerritSummaryCB(buildInfoList, results, status, arg):
                 })
 
 c['buildbotURL'] = 'http://buildbot.example.com/'
+
+# The 'GerritStatusPush' target is used to publish build results in Gerrit
+# To create message (with Code-Review) used function 'gerritReviewCB'
+# Notification about this message will be sent to to all (change owners,
+# reviewers, watchers and any user who has starred the change)
+# https://gerrit-documentation.storage.googleapis.com/Documentation/2.9/cmd-review.html#_options
+
 c['status'].append(status.GerritStatusPush(gerrit_url, gerrit_user,
+                                           notify='ALL',
```

Comment:
Hmm...  I checked https://gerrit-documentation.storage.googleapis.com/Documentation/2.11/cmd-review.html and it does not say anything either, however from my personal experience, at least, the owner does get the e-mail.